TOOLS/lua/autocrop.lua

```diff
@@ -29,7 +29,7 @@ require 'mp.options'
 
 
 -- auto: bool - Whether to automatically apply crop at the start of playback.
--- auto_delay: seconds - Delay before starting crop, only effective when playback time is less than this time.
+-- auto_delay: seconds - Delay before starting crop, only effective when playback time is less than this.
```

Comment:
Is it necessary to keep auto_delay?
detect_limit = 16 can cope with the black scene fadein

Reply:
I think it's probably still necessary, because I'm sure you can find some content with a longer pause or slower fade in.